internal/api/courses/routes.go

```diff
@@ -0,0 +1,18 @@
+package courses
+
+// All routes handled by this package.
```

Comment:
Maybe say "All the API endpoints handled by this package." since that's what the other files say.